src/graphql.ts

```diff
@@ -50,6 +50,11 @@ export class AuthUserInput {
     password: string;
 }
 
+export class ConfirmationCodeInput {
+    code: string;
+    email: string;
+}
+
 export class Album {
     __typename?: 'Album';
```

Comment:
This file is generated from the .graphql files, i will probably remove this so the diffs don't muddy up the PRs. Just ignore this file for now